### ISSUE TITLE
File disappearance[BUG] 

### ISSUE THREAD

**Issue by lIN02843** [BigSoftVideo/DOTE#16]
**Describe the bug**
A transcript folder, along with the files within it, disappeared after I tried to overwrite it by importing a .dote file, which might also appears to have an error. 

**To Reproduce**
_Steps to reproduce the behavior:_
_1. go to project folder
_2. open a transcript file
_3. import a .dote file
_4. sees error, and subsequently the entire transcript file disappears from local folder

**Expected behavior**
Recover the lost file.

**Screenshots**
<img width="357" alt="Screenshot 2024-01-31 at 15 01 48" src="https://github.com/BigSoftVideo/DOTE/assets/158387137/beaf31c8-d132-42c6-a897-79f92c17f806">
![Uploading Screenshot 2024-01-31 at 15.01.48.png…]()

**O/S and Hardware (please complete the following information):**
 - macOS M1
 - DOTE 1.0.5
 - .mov

**Additional context**
The .dote file I used to overwrite the original file might have been compromised itself for some reason.

**Comment by CEdwards74** [comment]
I have the same issue part way through importing an audio file

**Comment by LiminalSpaces** [comment]
> I have the same issue part way through importing an audio file

The original post was over two years ago on a much earlier version of DOTE. Also the original post was regarding importing a packaged Transcript, and it sounds like you are having trouble importing an audio file into an existing project?

Could you please provide more details regarding the problem you are having?

**Comment by CEdwards74** [comment]
Yes, I have set up a new project and am importing and audio file MP3. At around the 30% mark of it importing, the screen just goes blank and I cannot navigate anywhere. When I close and reopen DOTE it goes straight into the import again and the same thing happens.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by CEdwards74** [comment]
I have figured out the issue. I needed to install the ffmpeg wave-form generator in the settings. This seems to have fixed the problem.

Thanks,

Caroline
________________________________
